### PR TITLE
feature/rst-1122 Add static 404 page and assets

### PR DESCRIPTION
**Pull request by Stivaros**
Recreate 404 page using static HTML and static assets in the `public` folder.

### CODE REVIEW

File: public/404.html [hunk 1]
```diff
@@ -1,67 +1,135 @@
...
-    color: #666;
-    box-shadow: 0 3px 8px rgba(50, 50, 50, 0.17);
-  }
-  </style>
+  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
+  <meta charset="utf-8">
+  <title>Page Not Found - Response to Claim - MoJ</title>
+
+  <!--[if gt IE 8]><!-->
+  <link rel="stylesheet" media="screen" href="404_files/govuk-template.css" crossorigin="anonymous">
```
Comment (garytaylor) [inline public/404.html]: why special assets ? can you not just link to pre compiled assets in their standard location ?

File: public/404.html [hunk 2]
```diff
@@ -1,67 +1,135 @@
...
+  <!--<![endif]-->
+
+  <meta name="theme-color" content="#0b0c0c">
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <link rel="stylesheet" media="all" href="404_files/confirmation_supplied_details.css" data-turbolinks-track="reload">
+  <link rel="stylesheet" media="all" href="404_files/elements.css" data-turbolinks-track="reload">
+  <link rel="stylesheet" media="all" href="404_files/application.css" data-turbolinks-track="reload">
+  <script src="404_files/jquery.js"></script>
```
Comment (garytaylor) [inline public/404.html]: Do we need many of these ?  we dont need turbolinks, show-hide-content, rails-ujs, cation-cable or even application i doubt ?

File: public/404.html (hunk 1 above)
Comment (Stivaros) [reply public/404.html]: Fixed 81a7f01

Comment (Stivaros) [reply public/404.html]: Fixed in 81a7f01

File: public/404.html (hunk 2 above)
Comment (Stivaros) [reply public/404.html]: Removed in 81a7f01

**Pull request merged**